### PR TITLE
update pcloud doc

### PR DESCRIPTION
**Pull request by marsjane**
#### What is the purpose of this change?

update pcloud doc to avoid puzzling token error when use remote rclone authorize

#### Was the change discussed in an issue or in the forum before?

No. Just I met this when use the doc and want to save other ppl's time

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/pcloud.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "pCloud"
 description: "Rclone docs for pCloud"
-versionIntroduced: "v1.39"
+versionIntroduced: "v1.40"
```
Comment (albertony) [inline docs/content/pcloud.md:4]: Why this change?

**Review by albertony — COMMENTED** [review]
Thanks. Looks good to me, except I have one question.

File: docs/content/pcloud.md (hunk 1 above)
Comment (marsjane) [reply docs/content/pcloud.md:4]: aha sry I just thought I may need to update version if it has a new update, let me revert this part back

**Review by albertony — APPROVED** [review]

**Comment by marsjane** [comment]
> I think the should have happened automatically. Did you cut and paste the rclone authorise command along with the base64 characters when you ran it? Those characters should include the hostname.
> 
> It is possible this is broken - it is supposed to work without manual editing of the config
Yes I understand it is totally automatic when I set it up on a normal PC, I tried it do set up all correctly.
But the another case is I config the rclone on a nas, which has no website browser, so I connect to it via ssh to config it, when it ask for the token config, I just run the rclone authorise command on another pc and copy back the shown token config, sth like: (there is no hostname in it)
```
Paste the following into your remote machine --->
{"access_token":"","token_type":"","expiry":""}
<---End paste
```
In this case if you don't set the hostname manually you will get token error.

**Review by ncw — APPROVED** [review]
Yes you are right @marsjane 

I will merge this - thank you :-)
